### PR TITLE
Integrate toolbox with tether

### LINKED ISSUE #407: Add IP address reporting
vSphere relies on guest tools to report the IP addresses of a VM - we do not want to have the full guest tools installed as it is way too large, but we do want the option to report and IP address that will show up in the standard location in vSphere.

This is likely to be implemented in the same library as rpctool as I believe they rely on the same communication mechanism

### PR DESCRIPTION
**Pull request by dougm**
The toolbox service is started as a tether extension.
The VCH and container VM IP addresses are now displayed in WebClient and
via 'govc vm.ip'.

Fixes #407

### CODE REVIEW

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

File: infra/machines/vcsa/create-esxi-vm.sh [hunk 1]
```diff
@@ -17,19 +17,22 @@
...
 }
 
 disk=48
 mem=16
 iso=VMware-VMvisor-6.0.0-3634798.x86_64.iso
-vib=http://download3.vmware.com/software/vmw-tools/esxui/esxui-signed-3843236.vib
+vib=http://download3.vmware.com/software/vmw-tools/esxui/esxui-signed-3976049.vib
 
-while getopts c:d:i:m:s flag
+while getopts c:d:hi:m:s flag
```
Comment (corrieb) [inline infra/machines/vcsa/create-esxi-vm.sh]: shouldn't there be a colon between h and i?

**Comment by corrieb** [comment]
LGTM

File: infra/machines/vcsa/create-esxi-vm.sh (hunk 1 above)
Comment (dougm) [inline infra/machines/vcsa/create-esxi-vm.sh]: The colon is only for flags that take an argument, '-h' and '-s' do not take an argument.

File: infra/machines/vcsa/create-esxi-vm.sh [hunk 2]
```diff
@@ -17,19 +17,22 @@
...
+
+GOVC_* environment variables also apply, see https://github.com/vmware/govmomi/tree/master/govc#usage
+If GOVC_USERNAME is set, it is used to create an account on the ESX vm.  Default is to use the existing root account.
+If GOVC_PASSWORD is set, the account password will be set to this value.  Default is to use the given ESX_URL password.
+EOF
 }
 
 disk=48
 mem=16
 iso=VMware-VMvisor-6.0.0-3634798.x86_64.iso
```
Comment (caglar10ur) [inline infra/machines/vcsa/create-esxi-vm.sh]: should we also switch VMware-VMvisor-6.0.0-3620759.x86_64.iso while at it?

Comment (dougm) [inline infra/machines/vcsa/create-esxi-vm.sh]: yes, will do.

**Comment by dougm** [comment]
👊

**Pull request merged**